docs/introduction/framework-extensibility.md

```diff
@@ -0,0 +1,31 @@
+# Framework Extensibility
+
+This article summarizes current possibilities of the framework extension,
+provides a list of customizations that are not achievable now but are planned to be enabled soon, 
+as well as a list of customizations that are not (and will not be) possible at all.
+
+## What Is Achievable Easily
+* [Extending an entity](/docs/wip_glassbox/entity-extension.md)
+    * [Adding a new attribute](/docs/cookbook/adding-new-attribute-to-an-entity.md)
+* [Extending particular forms in the administration](/docs/wip_glassbox/form-extension.md) without the need of the template overriding
+* [Customizing database migrations](/docs/introduction/database-migrations.md)
+    * adding a new migration as well as skipping and reordering the existing ones
+* Configuring the smoke tests (see [`RouteConfigCustomization`](/project-base/tests/ShopBundle/Smoke/Http/RouteConfigCustomization.php) class)
+    * *Note: This is now achievable as the configuration class is located in the open box project-base. 
+    However, that makes the upgrading of the component harder so the configuration is planned to be re-worked yet*
```

Comment:
**yet** doesn't fit at the end of the sentence, I would just remove it. Also, there is a dot missing at the end.